arrays/arrays.c

```diff
@@ -21,30 +21,37 @@ typedef struct Array {
  *****/
 Array *create_array (int capacity) {
   // Allocate memory for the Array struct
-
+  Array *array = malloc(sizeof(Array));
   // Set initial values for capacity and count
-
+  array->capacity = cap;
+  array->count = 0;
   // Allocate memory for elements
-
+  array->elements = calloc(cap, sizeof(char *));
```

Comment:
Very good of you to use calloc() here but again "cap" is not defined anywhere in this function.